.github/workflows/ci.yml

```diff
@@ -0,0 +1,53 @@
+name: CI
+
+on:
+  pull_request:
+  push:
+    branches:
+      - main
+      - latest
+
+jobs:
+  build:
+    strategy:
+      fail-fast: false
+      matrix:
+        node-version: [10.0.x, 10.x, 12.0.x, 12.x, 14.0.x, 14.x, 15.x, 16.x]
```

Comment:
I feel like it's time to drop support for v10
it don't get security updates anymore

Reply:
Unfortunately we have to follow the `npm cli` on this for now.  When we drop `v10` there we can cascade that change down to all of its dependencies.

Reply:
As it happens, we will be discussing this very topic at today's RFC meeting! https://github.com/npm/rfcs/issues/444